src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/mobileclient_stack.py

```diff
@@ -1,48 +1,75 @@
+from typing import Optional
+
 from aws_cdk import aws_cognito, aws_iam, aws_s3, core, custom_resources
 
 from common.auth_utils import construct_identity_pool
 from common.common_stack import CommonStack
 from common.platforms import Platform
 from common.region_aware_stack import RegionAwareStack
+from common.secrets_manager import get_integ_tests_secrets
 
 
 class MobileClientStack(RegionAwareStack):
     def __init__(self, scope: core.Construct, id: str, common_stack: CommonStack, **kwargs) -> None:
 
         super().__init__(scope, id, **kwargs)
 
-        self._parameters_to_save["email_address"] = "aws-mobile-sdk-dev+mc-integ-tests@amazon.com"
-
         self._supported_in_region = self.are_services_supported_in_region(
             ["cognito-identity", "cognito-idp"]
         )
 
+        if not self._supported_in_region:
+            return
```

Comment:
Should we return silently or throw a warning to let the user know?

Reply:
If the other stacks are not handling this case, might be easier to have a common handler method to say:
"<StackName> requires <these> services, that are not supported in <this> region"